classes/Project/ProjectFactory.js

```diff
@@ -216,6 +216,71 @@ export default class ProjectFactory {
         message: "No image found. Cannot process further."
       }
     }
+
+    let isIIFImage = false
+    let IIIFServiceParts = imageURL.split('/').reverse()
+    let IIIFServiceJson = null
+
+    function isValidIIIFRegion(region) {
+      return (
+        region === "full" ||
+        region === "square" ||
+        /^pct:\d+(\.\d+)?,\d+(\.\d+)?,\d+(\.\d+)?,\d+(\.\d+)?$/.test(region) || 
+        /^\d+,\d+,\d+,\d+$/.test(region)
+      )
+    }
+
+    function isValidIIIFSize(size) {
+      return (
+        size === "max" ||
```

Comment:
in a 2.0 image API, "max" might be called "full". You can see why they changed it to avoid confusion.